src/utils/codec.ts

```diff
@@ -11,11 +11,12 @@ import { BASE1024_SCHEMA } from './constants'
 interface ICodecUtil {
```

Comment:
use `class` definition the object.

**ALL STATIC METHOD**

not expose any method, not require `_` prefix